src/public/Disable-ADSIComputer.ps1

```diff
@@ -88,6 +90,8 @@ function Disable-ADSIComputer
                 $Account = Get-ADSIComputer -Identity $Identity @ContextSplatting
                 $Account.enabled = $false
                 $Account.Save()
+                Write-Verbose "[$FunctionName] The Account $Identity was disable"
```

Comment:
```suggestion
                Write-Verbose "[$FunctionName] The Computer $Identity was disabled"
```

Technically correct but i think when talking about a account the most Admins think about a User account rather then a Computer account. So i suggest to specify the type of the Account.
+ fixed Typo